### DISCUSSION TITLE
关于我的开源项目使用到 Remix Icon 是否需要授权的问题

### DISCUSSION THREAD

**Discussion by dufu1991** [Remix-Design/RemixIcon#639] [Q&A] [answered]
我的开源项目 https://github.com/dufu1991/stdf 里面使用到了部分 Remix Icon，我在站点和仓库文档里面也有提及和感谢。但是今天才注意到 Remix Icon 的协议是 Apache License 2.0，没有 MIT 那么宽松，这里特意问一下，需不需要我做什么处理？

**Comment by xiaochunjimmy** [comment, marked as answer]
没问题的，不用特殊处理。
